composer.json

```diff
@@ -60,7 +60,8 @@
         "symfony/process": "3.4.2"
     },
     "require-dev": {
-        "phpunit/phpunit": "^7.5"
+        "phpunit/phpunit": "^7.5",
+        "ext-bcmath": "*"
```

Comment:
Infection itself should not depend on `ext-bcmath`. We don't use any of those functions.

Reply:
IMO `function_exists` should not be here, for one, because there's no actual need for it, and for another, Infection can be run under PHP without bcmath, where tested code can have it loaded dynamically. Best if we make no assumtions about missing or present modules: if there's a function in code, and we can mutate it, we should.

Reply:
That was leftover from testing - removed